tests/baselines/reference/contextualOverloadListFromArrayUnion.errors.txt

```diff
@@ -0,0 +1,69 @@
+three.ts(28,14): error TS2774: This condition will always return true since this function is always defined. Did you mean to call it instead?
+
+
+==== one.ts (0 errors) ====
+    declare const y: never[] | string[];
+    export const yThen = y.map(item => item.length);
+==== two.ts (0 errors) ====
+    declare const y: number[][] | string[];
+    export const yThen = y.map(item => item.length);
+==== three.ts (1 errors) ====
+    // #42504
+    interface ResizeObserverCallback {
+        (entries: ResizeObserverEntry[], observer: ResizeObserver): void;
+    }
+    interface ResizeObserverCallback { // duplicate for effect
+        (entries: ResizeObserverEntry[], observer: ResizeObserver): void;
+    }
+    
+    const resizeObserver = new ResizeObserver(([entry]) => {
+        entry
+    });
+    // comment in #35501
+    interface Callback<T> {
+        (error: null, result: T): unknown
+        (error: Error, result: null): unknown
+    }
+    
+    interface Task<T> {
+        (callback: Callback<T>): unknown
+    }
+    
+    export function series<T>(tasks: Task<T>[], callback: Callback<T[]>): void {
+        let index = 0
+        let results: T[] = []
+    
+        function next() {
+            let task = tasks[index]
+            if (!task) {

```

Comment:
Looks like might also have to bail if the source is accessing something from an array 🤔